### PR TITLE
Incorporating existing concept topics

### PR DESCRIPTION
**Pull request by digitaljoshua**
Incorporating any content in the existing concept topics that doesn't appear in the Reorg TOC branch.

I examined the content found in the 3 existing concept topics (introduction-to-data-storage.md, introduction-to-preconfigured-roles, introduction-to-account-management) for information that doesn't exist in the lasato-ocs_docs_reorg branch topics. I added the information into suitable topics within the lasato-ocs_docs_reorg branch.
The reason is that the 3 existing concept topics are going away and we want to preserve relevant information to the customer.

### CODE REVIEW

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ A user is an individual identity that represents a person using OCS. When users
 
 Clients have programmatic access to OCS resources through OCS APIs. There are two primary client types: 
 
-* **Client credential clients** – The most common type of OCS client, used for server-to-server communication without the presence or intervention of a user. Examples include PI Adapters or the Edge Data Store sending data to OCS. This type of client is issued a client ID and secret. After authentication, the client is granted an access token with a defined lifetime. 
+* **Client credential clients** – The most common type of OCS client, used for server-to-server communication without the presence or intervention of a user. Examples include PI Adapters or the Edge Data Store sending data to OCS. This type of client is issued a client ID and secret. After authentication, the client is granted an access token with a defined lifetime. The tokens may either be short-lived access tokens or longer-lived refresh tokens that allow the client to request new access tokens.
```
Comment (dcbroad3) [inline Content_Portal/Overview/what-does-ocs-do.md:44]: I'm not sure what is meant here. There isn't an either/or option for the tokens: you can specify any duration for the token lifetime, and refresh tokens to request a new access token are always included regardless of the token duration.

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Authorization is the process of determining the appropriate access level for a u
 
 ### Access control list 
 
-Each OCS service and resource has an access control list (ACL) that defines how much access is granted to assigned roles. The OCS Administrator configures each ACL and specifies types of permissions for each role. When a request is made to a specific OCS resource, the role assigned to the requestor (whether a user or client) is compared to the ACL for that resource to determine whether the request should be authorized. 
+Each OCS service and resource has an access control list (ACL) that defines how much access is granted to assigned roles. The OCS Administrator configures each ACL and specifies types of permissions for each role. When a request is made to a specific OCS resource, the role assigned to the requestor (whether a user or client) is compared to the ACL for that resource to determine whether the request should be authorized. Users are granted access permissions to OCS objects based on their assigned roles and the corresponding ACLs.
```
Comment (dcbroad3) [inline Content_Portal/Overview/what-does-ocs-do.md:68]: You might want to consider saying "granted or denied access" or "allowed or denied access". This wording is OK though, if you just want to be more generalized.

File: Content_Portal/Overview/what-does-ocs-do.md (hunk 1 above)
Comment (digitaljoshua) [reply Content_Portal/Overview/what-does-ocs-do.md]: I agree with this assessment. I just am going to remove this added sentence.

File: Content_Portal/Overview/what-does-ocs-do.md (hunk 2 above)
Comment (digitaljoshua) [reply Content_Portal/Overview/what-does-ocs-do.md]: changed as suggested.

**Pull request merged**